packages/tools/guiEditor/src/components/commandBarComponent.tsx

```diff
@@ -338,9 +337,7 @@ export class CommandBarComponent extends React.Component<ICommandBarComponentPro
                         )}
                     </div>
                 </div>
-                <div className="commands-right">
-                    <img src={betaFlag} className="beta-flag" draggable={false} />
-                </div>
+                <div className="commands-right"></div>

```

Comment:
Should we still keep it ?

Reply:
I thought to keep it if we needed to put anything later and also to not risk messing with the existing CSS :P